### ISSUE TITLE
laravel 6

### ISSUE THREAD

**Issue by scripta55** [nicolaslopezj/searchable#200]
hi, just curious to know if you will have an update for laravel 6? thanks

**Comment by leknoppix** [comment]
Hi,
I have try to update to laravel 6 but, I can't.
I have try to a new project build in laravel 6, I have error.

**Comment by scripta55** [comment]
there is a pull request pending, someone has already done it, we are just
waiting on nico to accept :)

Kind Regards,

Peter Phillip
+27 72 900 6987 (South Africa)
+265 99 566 3635 (Malawi)
+258 84 767 1178 (Mozambique)
+27 21 813 9934 (International Roaming)
*Nothing will work unless you do.*

**Comment by JhumanJ** [comment]
So PR was merged, but new version isn't released (on packaggist at least)... What can we do about it ?

**Comment by nicolaslopezj** [comment]
I can see the version 1.12.0 published in packagist

![image](https://user-images.githubusercontent.com/2042567/66654282-48694080-ec10-11e9-8647-e14635d84730.png)

**Comment by JhumanJ** [comment]
My bad ! That was not the case yesterday so I couldn't complete my migration to L6.
